src/basic/inotify-util.h

```diff
@@ -31,7 +31,7 @@
 
 union inotify_event_buffer {
         struct inotify_event ev;
-        uint8_t raw[INOTIFY_EVENT_MAX];
+        uint8_t raw[INOTIFY_EVENT_MAX] _alignas_(struct inotify_event);
```

Comment:
this is redundant. This is a union anyway, and unions have the largest alignment of all its components and one component is "struct inotitfy_event" anyway. Hence this changes exactly nothing.